### PR TITLE
Update 2022-12-14-sams-foundry-experience.md

### PR DESCRIPTION
**Pull request by lbenezra-FA**
## Description & motivation

Found a few grammar bugs to fix! Let me know if you have questions. Lovely read!

### CODE REVIEW

File: website/blog/2022-12-14-sams-foundry-experience.md [hunk 1]
```diff
@@ -12,48 +12,48 @@ date: 2022-12-15
...
+> The [Foundry Program](https://www.getdbt.com/blog/announcing-the-foundry-program/) is an apprenticeship designed to turn data newbies into fully-fledged analytics engineers over the course of six months. As one of the inaugural Foundry apprentices, I’m here to share my journey into analytics engineering, along with the takeaways I picked up along the way. We’re continuing to improve the program with each iteration, but the curriculum for my cohort was split into two parts—three months of training followed by three months of hands-on work.
 
 ### Getting started
 
-So as I got started with the apprenticeship, did my expectations play out into reality? Of course, the answer to that is going to be “yes and no”, but maybe not for reasons you expect.
+As I got started with the apprenticeship, did my expectations play out into reality? The answer to that is "yes and no," but maybe not for reasons you expect.
```
Comment (nghi-ly) [inline website/blog/2022-12-14-sams-foundry-experience.md]:
```suggestion
As I got started with the apprenticeship, did my expectations play out into reality? The answer to that is yes and no, but maybe not for reasons you expect.
```

**Review by nghi-ly — COMMENTED** [review]
thank you so much for making these updates, @lbenezra-FA ! left one sugg, a nit

**Review by SamHarting — APPROVED** [review]
looks good to me! thanks!

**Review by runleonarun — APPROVED** [review]
Looks like we can merge this now! Thank you for adding these improvements!

**Pull request merged**